moped-editor/src/views/projects/projectView/ProjectComponents/EditAttributesModal.js

```diff
@@ -53,8 +53,8 @@ const EditAttributesModal = ({
     const { subcomponents, phase, subphase, tags } = formData;
     const completionDate = !!phase ? formData.completionDate : null;
     const description =
-      formData.description.length > 0 ? formData.description : null;
-    const srtsId = formData.srtsId.length > 0 ? formData.srtsId : null;
+      formData.description?.length > 0 ? formData.description : null;
+    const srtsId = formData.srtsId?.length > 0 ? formData.srtsId : null;
```

Comment:
The form is now returning `null` when these are blank so we need to check for this. Seems like these length checks can be removed in the form refactor since it will pass along `null` to the DB without special handling.